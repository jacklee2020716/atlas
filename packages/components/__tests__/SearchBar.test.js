import React from "react"
import { mount } from "enzyme"
import SearchBar from "./../src/components/SearchBar"

describe("SearchBar component", () => {

  const component = mount(<SearchBar placeholder="Type here..." value="test" />)

  it("Should render correctly", () => {
    expect(component).toMatchSnapshot()
  })

})