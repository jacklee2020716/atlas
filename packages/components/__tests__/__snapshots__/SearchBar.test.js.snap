// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component Should render correctly 1`] = `
<Memo(SearchBar)
  placeholder="Type here..."
  value="test"
>
  <div
    css={
      Object {
        "map": undefined,
        "name": "laoupi",
        "next": undefined,
        "styles": "
      display: flex;
      padding: 1.5rem;
      margin: 1rem;
      border-radius: 0.5rem;
      border: none;

      & * {
        font-size: 1rem;
      }
    ",
        "toString": [Function],
      }
    }
  >
    <input
      css={
        Object {
          "map": undefined,
          "name": "c0kzty",
          "next": undefined,
          "styles": "
      padding: 0.75rem 0.5rem;
      font-weight: 100;
      border-radius: 0.25rem;
      border: 1px solid #000;
    ",
          "toString": [Function],
        }
      }
      onChange={[Function]}
      placeholder="Type here..."
      type="text"
      value="test"
    />
    <Memo(Button)
      onClick={[Function]}
    >
      <button
        css={
          Object {
            "map": undefined,
            "name": "1s1uxvy",
            "next": undefined,
            "styles": "
    
    padding: 0.75rem 0.5rem;

    font-size: 1rem;

    width: auto;
  ;
    border-radius: 0.25rem;
    border-color: ;
    border-style: solid;
    border-width: 2px;
    font-style: 'Lato','Helvetica Neue',Arial,Helvetica,sans-serif;
    background-color: #2578C2;
    color: #fff;
  ",
            "toString": [Function],
          }
        }
        onClick={[Function]}
      >
        Search
      </button>
    </Memo(Button)>
  </div>
</Memo(SearchBar)>
`;
